apps/balancer-tools/src/app/poolsimulator/(components)/AnalysisPage.tsx

```diff
@@ -1,6 +1,5 @@
 "use client";
```

Comment:
Should we move this to the `index` page file now that all the app is on a single page?

Reply:
I would say this is still a component. It would be too big to be on `index` IMO